mdnDocsOverrides.json

```
@@ -149,14 +171,18 @@
   "atomics-wait-async": [
     "Web/JavaScript/Reference/Global_Objects/Atomics/waitAsync"
   ],
-  "attr": [],
+  "attr": [
+    "Web/CSS/attr"
+  ],
   "attr-contents": [
     "Web/CSS/attr"
   ],
   "audio": [
     "Web/HTML/Element/audio"
   ],
-  "audio-session": [],
+  "audio-session": [
+    "Web/API/Navigator"
+  ],
```

This feature is also not documented on MDN. I feel like it's better to leave this one empty rather than point to the page about `navigator`. At least for now.

```suggestion
  "audio-session": [],
```